app/views/organisation_relationships/add_managing_agent.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= form_with model: @organisation, url: managing_agents_organisation_path, method: "post", local: true do |f| %>
+  <% if current_user.support? %>
+    <%= render partial: "organisations/headings", locals: { main: @organisation.name, sub: nil } %>
+    <%= render SubNavigationComponent.new(items: secondary_items(request.path, @organisation.id)) %>
+    <h2 class="govuk-visually-hidden">Add Managing Agent</h2>
+  <% end %>
```

Comment:
I think lines 6 and 7 can be removed, just noticed on the equivalent housing providers page too.

Reply:
I'll update both 👍